### DISCUSSION TITLE
How can we get more people to use digital coins more often than fiat?

### DISCUSSION THREAD

**Discussion by tktkowash** [JasmyCoin/JasmyCoin#24] [General]
How can we get more people to use digital coins more often than fiat?

**Comment by harayamada** [comment]
＞＞the point of view from payee 
・No need to go to the bank
・No fees to the bank
・No shortage of change
・Customer information can be converted into data
・Customers' shopping history can be used.
・Less time spent on checkout

**Comment by harayamada** [comment]
＞＞ the point of view from payers
If they can buy things more cheaply with coin than with fiat・・・, 
the number of times that they use coin for transactions should also increase.it contributes to the circulation speed of the coin.
A bicycle shop crews’ impression: "Customers feel that they will get a portion of the payment amount back when they use digital coins”

**Reply by harayamada** [reply]
Another thing is that the validity of the Shopping Points have time-limited.
「I've been wondering, "If I sell an item on Mercalli, why is there an expiration date, and if I don't use it by then, why is it confiscated? I have wondered」

**Reply by harayamada** [reply]
In Japan, under the regulations of the Law Concerning Settlement of Funds（資金決済法）、Point issures  have to deposite money on authorities for consumers.
Point issuers are trying to minimize the amount of deposit money required for consumer protection.
In order to keep the deposit for consumer protection as low as possible, Point issuers have set a time limit on the points they issue.

However, I don't think this is the only reason. I think it's not just that, because setting the expiration date hastens the use of the coin.
In other words, it contributes to the circulation speed of the coin

**Reply by harayamada** [reply]
digital coins has the character of disappearing （Or diminished ）values  after the due day contracted

It seems appropriate from the perspective of accelerating currency velocity. This could be implemented with smart contracts.
The coin will age.
